.gitlab-ci.yml

```diff
@@ -81,8 +81,8 @@ build-linux:
   <<:                              *collect-artifacts
   <<:                              *build-refs
   script:
-    - time cargo build --release
-    - time cargo test --release --all
```

Comment:
what was wrong with it?

Reply:
I thought this was what you meant by `just remove the debug lines when you're done` :-) ?

Reply:
it's mostly needed for `cargo test` because no one knows how much the tests take up.